release/docker/build-image-multi-arch.sh

```diff
@@ -84,7 +84,7 @@ if [ -z "$VERSION" ] || [ -z "$DOCKERFILE" ] || [ -z "$PRODUCT" ] || [ -z "$ARCH
   exit 1
 else
   echo $VERSION $DOCKERFILE $PRODUCT $ARCHITECTURE
-  IFS=', ' read -r -a ARCHITECTURE_ARRAY <<< "$ARCHITECTURE"
+  IFS=',' read -r -a ARCHITECTURE_ARRAY <<< "$ARCHITECTURE"
   IFS=', ' read -r -a TARBALL_ARRAY <<< "$TARBALL"
```

Comment:
Does this also need to be fixed? Space being not a valid separator.

Or is the above change not needed to allow for it?

Reply:
yes you're right @dblock good catch!